pkg/ui/react-app/src/thanos/pages/blocks/Blocks.tsx

```diff
@@ -55,38 +64,47 @@ export const BlocksContent: FC<{ data: BlockListProps }> = ({ data }) => {
   };
 
   if (err) return <UncontrolledAlert color="danger">{err.toString()}</UncontrolledAlert>;
-
+  const date = new Date(refreshedAt).toLocaleString().split(',');
   return (
-    <>
+    <div>
```

Comment:
Minor nit. This `div` seems redundant, as we essentially have
```html
<div>
  <div>
    {........}
  </div>
</div>
```
So the outer div is just creating an unnecessary div soup.